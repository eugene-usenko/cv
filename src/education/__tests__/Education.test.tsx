import { createRenderer, ShallowRenderer } from 'react-test-renderer/shallow'
import { Education } from '..'

describe('Education', () => {
    const shallow: ShallowRenderer = createRenderer()

    beforeEach(() => {
        jest.resetAllMocks()
    })

    it('render', () => {
        const a = shallow.render(<Education />)
        expect(a).toMatchSnapshot()
    })
})
