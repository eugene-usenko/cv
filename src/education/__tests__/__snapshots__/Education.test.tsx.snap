// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Education render 1`] = `
<Stack>
  <SubTitle
    uppercase={true}
  >
    cv.education
  </SubTitle>
  <ul>
    <li>
      <StyledLinkBase
        href="http://pgasa.dp.ua/en/"
      >
        PSACEA: Automation of Technological & Manufacturing Processes
      </StyledLinkBase>
    </li>
    <li>
      <StyledLinkBase
        href="https://github.com/eugene-usenko/cv/blob/main/doc/1Z0-851.pdf"
      >
        Java OCJP 1.6(1Z0-851)
      </StyledLinkBase>
    </li>
    <li>
      <StyledLinkBase
        href="http://university.mongodb.com/course_completion/1ad65bdb-de01-42b0-8fa8-5de09f83"
      >
        M101JS: MongoDB for Node.js Developers
      </StyledLinkBase>
    </li>
  </ul>
</Stack>
`;
